patterns/hidden-404.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Title: 404
+ * Slug: twentytwentyfive/hidden-404
+ * Inserter: no
+ *
+ * @package TwentyTwentyFive
+ */
+
+?>
+
+<!-- wp:columns {"style":{"spacing":{"padding":{"top":"50px","bottom":"50px","left":"50px","right":"50px"},"blockGap":{"left":"22px"}}}} -->
```

Comment:
It looks like the values are not using the presets here yet, and it's breaking the editor. We may also want to wrap the columns in another group to use full-width alignment.